plugins/woocommerce/src/Blocks/BlockTypes/MiniCartFooterBlock.php

```diff
@@ -14,6 +14,16 @@ class MiniCartFooterBlock extends AbstractInnerBlock {
 	 */
 	protected $block_name = 'mini-cart-footer-block';
 
+	/**
+	 * Enqueue frontend data.
+	 *
+	 * @param array $attributes Block attributes.
+	 */
+	protected function enqueue_data( array $attributes = array() ) {
+		parent::enqueue_data( $attributes );
+		$this->asset_data_registry->add( 'miniCartFooterDescription', $this->get_total_items_description() );
```

Comment:
Does it make sense to render this variable if it's an empty string? Perhaps we can just check if `miniCartFooterDescription` exists, and if it doesn't, render nothing?

Reply:
Good call. See 0fe47dc